autoload/coverage.vim

```diff
@@ -85,7 +91,19 @@ endfunction
 " Hides coverage layer.
 function! s:CoverageHide() abort
   if has_key(s:visible, expand('%:p'))
-    execute 'sign unplace * file=' . expand('%:p')
+    " Unplace our signs for the current file.
+    let fname = bufname('%')
```

Comment:
Helpers for programmatically listing and unplacing signs would be great to have in maktaba (see https://github.com/google/vim-maktaba/issues/142). Consider splitting them out as standalone helper functions so they're easier to migrate over.